files/image_config/interfaces/interfaces.j2

```diff
@@ -88,6 +88,19 @@ iface {{ name }} {{ 'inet' if prefix | ipv4 else 'inet6' }} static
 {% for route in MGMT_INTERFACE[(name, prefix)]['forced_mgmt_routes'] %}
     up ip {{ '-4' if prefix | ipv4 else '-6' }} rule add pref {{ force_mgmt_route_priority }} to {{ route }} table {{ vrf_table }}
 {% endfor %}
+{% if SYSLOG_SERVER is defined and SYSLOG_SERVER %}
+{% for server in SYSLOG_SERVER %}
```

Comment:
Suggest adding indentation to improve readability.

Reply:
In this way, it will help format the output file. You can see the output test j2 file have the same indentation.